create_aws_application

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# this is the name of the application you want to create
+NAME=$1
+# these are all the environments. the buckets will be named NAME-ENVIRONMENT
+ENVIRONMENTS=(master develop testing)
+# region where you want to create the bucket
+REGION=eu-central-1
+# aws-cli profile to use (default default)
+PROFILE=${2:-default}
+# policy name (default UserBucketFullControl)
+USERS_POLICY_NAME=${3:-UserBucketFullControl}
```

Comment:
Wouldn't it be better to encourage the use of our default policy, `RenuoAppStandardPermissionsV2`?
The Amazon Policies will need to be reviewed, IMO